src/main/cli/process.ts

```diff
@@ -140,5 +140,24 @@ export function cli(mainFct: () => void) {
             },
         )
         .help()
-        .parse((_PACKAGING === "0") ? process.argv.slice(2) : process.argv.slice(1));
+        .parse(cliParse());
+}
+
+// arrow function to filter known option in CLI
+const knownOption = (str: string) => [
+    "--help",
+    "--version",
+].includes(str);
+
+function cliParse() {
+    // ./electron main.js --my-argument
+    process.argv.shift();
```

Comment:
I would strongly suggest **not** mutating the `process.argv` array.
For example:
```
return process.argv.slice((_PACKAGING === "0") ? 2 : 1).filter((arg) => arg === "--help" || arg === "--version" || !arg.startsWith("--"));
```

Reply:
Actually, I had a look at my old comment ( https://github.com/readium/readium-desktop/issues/547#issuecomment-520785137 ) and it reminded me that parameters can be injected first rather than last!
So, we need to filter first, then slice. Example code:
```
return process.argv.filter((arg) => arg === "--help" || arg === "--version" || !arg.startsWith("--")).slice((_PACKAGING === "0") ? 2 : 1);
```

Reply:
done https://github.com/readium/readium-desktop/pull/607/commits/a8f78c2ad87bacdd075de43bee237a8635598ee5